diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -58,6 +58,69 @@ int DifferentialFunctionComparator::cmpGEPs(const GEPOperator *GEPL,
         // The original function says the GEPs are equal - return the value
         return OriginalResult;
 
+    if (isa<ArrayType>(GEPL->getSourceElementType())
+        && isa<ArrayType>(GEPR->getSourceElementType())) {
+        // The number of elements of two enums can be different and
+        // its not considered as semantic change
```

Comment:
Better comment:
"When the only difference is the size of the accessed array, it is not considered a semantic change if the index type is an enum"